### PR TITLE
Moving to react native 0.30.0

### PR DESCRIPTION
**Pull request by wmathurin**

### CODE REVIEW

File: app/android/app/src/com/salesforce/samples/myuserpicreactnative/MyUserPicReactNativeApp.java [hunk 1]
```diff
@@ -28,14 +28,43 @@
...
 /**
  * Application class for our application.
  */
-public class MyUserPicReactNativeApp extends Application {
+public class MyUserPicReactNativeApp extends Application implements ReactApplication {
+
+	private final ReactNativeHost mReactNativeHost = new ReactNativeHost(this) {
+		@Override
+		protected boolean getUseDeveloperSupport() {
+			return BuildConfig.DEBUG;
```
Comment (bhariharan) [inline app/android/app/src/com/salesforce/samples/myuserpicreactnative/MyUserPicReactNativeApp.java:51]: Do we still need [this](https://github.com/wmathurin/MyUserPicReactNative/pull/10/files#diff-0e4af3785a71497126051fcd040f26faR83) in the `Activity`?

**Comment by bhariharan** [comment]
👍

File: app/android/app/src/com/salesforce/samples/myuserpicreactnative/MyUserPicReactNativeApp.java (hunk 1 above)
Comment (wmathurin) [inline app/android/app/src/com/salesforce/samples/myuserpicreactnative/MyUserPicReactNativeApp.java:51]: Good catch, we don't. Fixed here: https://github.com/wmathurin/MyUserPicReactNative/pull/10/commits/1378d9f27337354893ae5ac3c0c3d642cb82613a

Comment (bhariharan) [inline app/android/app/src/com/salesforce/samples/myuserpicreactnative/MyUserPicReactNativeApp.java:51]: 👍

**Pull request merged**